components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -76,10 +78,21 @@ function InputBox(props: InputBoxProps) {
     }
   }
 
+  const querySubmitted = React.useRef(false)
+
   const maybeAutofocus = (node: HTMLTextAreaElement | null) => {
-    if (node && props.context.selectedActionType) {
+    if (!node) {
+      return
+    }
+
+    if (props.context.selectedActionType) {
       node.focus()
     }
+    if (props.context.isMobile && props.context.hasAcceptedAgreement &&
```

Comment:
Hey, I think this would be quite useful on desktop too - when you open AIChat in the sidebar the input should be focused - I'm pretty sure some people in one of the slack chats were asking for it.

Maybe the `isMobile` check could just be for showing the soft keyboard?